source/includes/_themes_converting.md

```diff
@@ -0,0 +1,295 @@
+# Converting themes
+
+Earlier Mautic templates used PHP based on Symfony templates, however with the release of the 2.0 series, templates now use Twig.
```

Comment:
I've added a bit of clarification which I hope makes sense.  Basically you can (as far as I understand) still use the themes for existing resources but you won't be able to use them for new resources.